test/png.js

```diff
@@ -0,0 +1,18 @@
+var tape = require('tape');
+var mjAPI = require("..//lib/mj-single.js");
+tape('basic test: check MathJax core', function(t) {
+    t.plan(2);
+
+    var tex = '\\sin(x)';
+    mjAPI.start();
+
+    mjAPI.typeset({
+        math: tex,
+        format: "inline-TeX",
+        png: true,
+        mathoidStyle: true,
```

Comment:
It's not used in the test. So it can be removed. mathoidStyle is a modified version of img which only returns the style